src/IntervalFormatter.test.js

```diff
@@ -83,9 +83,13 @@ describe('IntervalFormatter', () => {
       .returns(unsubscriber)
```

Comment:
Maybe, but I think we should leave it to another PR since this one is basically to ensure the old timer is cleaned up independently of it being triggered or not